Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -4,19 +4,21 @@
 using System.IO;
 using System.Linq;
 using UnityEditor;
-using UnityEngine.InputSystem.Utilities;
 
 namespace UnityEngine.InputSystem.Editor
 {
     internal static class ProjectWideActionsAsset
     {
         internal const string kDefaultAssetPath = "Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsTemplate.inputactions";
-        internal const string kAssetPath = "ProjectSettings/InputManager.asset";
+        internal const string kInputManagerAssetPath = "ProjectSettings/InputManager.asset";
+        internal const string kAssetPath = "ProjectSettings/InputSystemActions.inputactions";
         internal const string kAssetName = InputSystem.kProjectWideActionsAssetName;
 
         static string s_DefaultAssetPath = kDefaultAssetPath;
         static string s_AssetPath = kAssetPath;
 
+        public static string assetPath => s_AssetPath;
```

Comment:
Is this writable for tests. 
Seems in most cases we want this read only.

Reply:
Yep, it can be readonly